dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -333,6 +351,39 @@ public void testAddVersionTagManualContainer() {
             count);
     }
 
+    @Test
+    public void testGrabChecksumFromDockerHub() {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        ContainersApi toolApi = new ContainersApi(webClient);
+        ContainertagsApi toolTagsApi = new ContainertagsApi(webClient);
+        DockstoreTool tool = createManualDockerHubTool();
+
+        tool.setDefaultDockerfilePath("/Dockerfile");
+        tool.setDefaultCwlPath("/Docstore.cwl");
+        tool = toolApi.registerManual(tool);
+
+        tool = addDockerHubTag(tool, toolTagsApi, toolApi);
+
+        List<Tag> tags = toolApi.getContainer(tool.getId(), null).getWorkflowVersions();
+        for (Tag tag: tags) {
+            assertNotNull(tag.getImages().get(0).getChecksums().get(0).getType());
```

Comment:
Being paranoid here, is it possible that the checksum retrieval code is failing and returning empty strings?